addons/sourcemod/scripting/zr_personal_skins.sp

```diff
@@ -497,3 +391,118 @@ stock void GetExtension(char[] path, char[] buffer, int size)
 	extpos++;
 	strcopy(buffer, size, path[extpos]);
 }
+
+stock void ResetClient(int client)
+{
+	g_bHasPersonalSkinsZombie[client] = false;
+	g_bHasPersonalSkinsHuman[client] = false;
+	g_sPlayerModelZombie[client] = "\0";
+	g_sPlayerModelHuman[client] = "\0";
+}
+
+stock void VerifyGroups()
```

Comment:
every function in here is exactly the same function but with one different parameter